Makefile

```diff
@@ -17,9 +17,13 @@ gofmt:
 	gofmt -w pkg/.. cmd/..
 
 e2e:
-	(cd e2e/bazel-5 && bazelisk build //...)
-	(cd e2e/bazel-6 && bazelisk build //...)
-	(cd e2e/bazel-7 && bazelisk build //...)
+	@for version in 5.x 6.x 7.x; do \
+		( \
+			cd e2e/bazel-workspace && \
+			echo "Testing $$version" > /dev/stderr && \
+			USE_BAZEL_VERSION=$$version bazel --batch build //...\
```

Comment:
should this be `bazelisk` instead of `bazel`?

Reply:
It's already bazelisk, no one uses bazel directly, and in most cases it's linked

Reply:
Yeah I do the same, but still, maybe we can keep it consistent in the file :)

Reply:
Ohh you mean the makefile uses bazelisk? Didn't notice, will check tomorrow and fix it

Reply:
fixed